encoders/src/test/java/au/csiro/pathling/encoders2/SchemaConverter2Test.java

```diff
@@ -0,0 +1,24 @@
+package au.csiro.pathling.encoders2;
+
+import au.csiro.pathling.encoders.AbstractSchemaConverterTest;
+import au.csiro.pathling.encoders.SchemaConverter;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class SchemaConverter2Test extends AbstractSchemaConverterTest {
+
+  @Override
+  protected SchemaConverter createSchemaConverter(int maxNestingLevel) {
+    return new SchemaConverter2(FHIR_CONTEXT, DATA_TYPE_MAPPINGS, maxNestingLevel);
+  }
+
+  // TODO: Complete when support for DirectChildResource e.g. in Parameters is added.
+  @Test
+  @Ignore
+  public void testNestedTypeInChoice() {
+
+    //  Parameters
+    //  ElementDefinition: parameter.valueElementDefinition-> parameter.valueElementDefinition.fixedElementDefinition
+    //  ElementDefinition: parameter.valueElementDefinition-> parameter.valueElementDefinition.example.valueElementDefinition (indirect)
+  }
```

Comment:
I think we should explicitly disallow import of Parameters.

Reply:
I have added the check together with other unsupported resources, but this is somewhat different issues and Parameters was just one of the Resources which it can be tested on. But we can also use Extension.
I have added more info on the issue and marker is to be addressed as part of [#414].